integration/e2e/db/db.go

```diff
@@ -16,38 +17,36 @@ const (
 )
 
 // NewMinio returns minio server, used as a local replacement for S3.
-func NewMinio(bktName string) *e2e.Service {
-	return e2e.NewService(
-		"minio",
+func NewMinio(port int, bktName string) *e2e.HTTPService {
+	return e2e.NewHTTPService(
+		fmt.Sprintf("minio-%v", port),
```

Comment:
Could you explain why we need to add the port to the service name for Minio, a part from `scenario_test.go`? Is it a requirement you have on the Thanos side?

Reply:
Because names has to be unique (container name)

Reply:
My question was to understand if this change (which we don't have for other services) is a requirement from the Thanos side or just for `scenario_test.go`.

Reply:
Just scenario, we indeed don't plan to run minio twice so far (however we might want to at some point when we introduce support for multiple buckets etc).

I think it's also a good practice for other services maybe? Is this a problem?

Reply:
Definitely not a problem. I was just wondering why the `NewMinio()` interface (which also requires the port) is different than other services.